### ISSUE TITLE
Pandas not installed as a dependency

### ISSUE THREAD

**Issue by benmaier** [pola-rs/polars#974]
#### Are you using Python or Rust?

Python.

#### Which feature gates did you use?

This can be ignored by Python users.

#### What version of polars are you using?

0.8.10

#### What operating system are you using polars on?

Ubuntu 20.04

#### Describe your bug.

When installing polars in a fresh Python environment, pandas is not installed as a dependency and thus importing polars fails.

#### What are the steps to reproduce the behavior?

In a fresh environment, install polars

```bash
$ pip install polars
Defaulting to user installation because normal site-packages is not writeable
Collecting polars
  Downloading polars-0.8.10-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (10.3 MB)
     |████████████████████████████████| 10.3 MB 4.6 MB/s
Collecting pyarrow==4.0.*
  Downloading pyarrow-4.0.1-cp38-cp38-manylinux2014_x86_64.whl (21.9 MB)
     |████████████████████████████████| 21.9 MB 12.1 MB/s
Requirement already satisfied: numpy in /usr/local/lib/python3.8/dist-packages (from polars) (1.19.2)
Installing collected packages: pyarrow, polars
Successfully installed polars-0.8.10 pyarrow-4.0.1
```

Afterwards, try to import polars

```python
In [1]: import polars as pl
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-5a5660e7af75> in <module>
----> 1 import polars as pl

~/.local/lib/python3.8/site-packages/polars/__init__.py in <module>
      1 # flake8: noqa
      2 from .series import Series, wrap_s
----> 3 from .frame import DataFrame, wrap_df, StringCache
      4 from .functions import *
      5 from .lazy import *

~/.local/lib/python3.8/site-packages/polars/frame.py in <module>
     41 from .utils import coerce_arrow, _is_expr, _process_null_values
     42 import polars
---> 43 import pandas as pd
     44 import pyarrow as pa
     45 import pyarrow.parquet

ModuleNotFoundError: No module named 'pandas'
```

#### What is the actual behavior?

shown above

#### What is the expected behavior?

What do you think polars should have done?

```python
In [1]: import polars

In [2]:
```

Edit: initially reported pyarrow version instead of polars version

**Comment by benmaier** [comment]
I just saw a commit that probably addressed this (bb2e8b83c2db80ad23df4941fa2dd778ab530335), so I'm guessing it's already being worked on :)

**Comment by ritchie46** [comment]
> I just saw a commit that probably addressed this (bb2e8b8), so I'm guessing it's already being worked on :)

Yes :slightly_smiling_face:  There is also a pre-release with a fix: https://pypi.org/project/polars/0.8.11_beta.1/

**Comment by benmaier** [comment]
thanks!
